TYPO3.Flow/Documentation/TheDefinitiveGuide/PartV/CodingGuideLines/PHP.rst

```diff
@@ -66,35 +66,30 @@ General considerations
 *The Flow standard file header*::
 
  <?php
- namespace YourCompany\Package\Stuff\Here;
+ namespace Neos\Flow\Something\New;
```

Comment:
Because otherwise the "standard header" doesn't match. We can reverse that, of course.